src/main/features/response/routes/view-response.ts

```diff
@@ -0,0 +1,19 @@
+import * as express from 'express'
+import { Paths } from 'response/paths'
+import { ErrorHandling } from 'common/errorHandling'
+import { Claim } from 'claims/models/claim'
+import { User } from 'idam/user'
+import { ClaimStoreClient } from 'claims/claimStoreClient'
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(
+    Paths.viewResponsePage.uri,
+    ErrorHandling.apply(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
```

Comment:
Please make sure that 404 is returned in case someone is accessing the page when the response has not been made yet. Simply response summary does not exists if response does not exist.